lib/analyzer/hash-analyzer.ts

```diff
@@ -0,0 +1,21 @@
+import { Docker } from "../docker";
+
+export { analyze };
+
+const HASH_PKGPATHS = ["**/node"];
```

Comment:
please add a manual test to make sure we're doing good when there are 2 different `node` binaries (not only in the scope of the plugin but also the rest of the system) - you can also do it after merging this PR :)

Reply:
Yes I need to add some tests. This PR is just the bare minimum to get it working ...